plaidml/bridge/openvino/ops/embedding_bag_packed_sum.cpp

```diff
@@ -32,7 +32,7 @@ static OpRegistration reg("EmbeddingBagPackedSum", [](const Context& ctx) {
     with_weights = true;
   }
 
-  auto I_gathered = gather(I, indices);
+  Tensor I_gathered = edsl::gather(I, indices);
```

Comment:
To be consistent, drop the `using namespace plaidml::edsl` above and use `edsl::Tensor` here.

Reply:
Fixed